src/shared/components/button-link/ButtonLink.scss

```diff
@@ -1,13 +1,14 @@
 @import 'src/styles/common';
 
 .buttonLink {
+  --_button-color: var(--button-link-color, #{$blue});
```

Comment:
How is this definition different from others for it to have and `_`?

Reply:
It isn't really different; it's just intentionally ugly not to be accidentally used in a parent 😄 It's like naming private fields in javascript before the private fields were introduced into the language. I could have simply used the `var(--button-link-color, #{$blue})`, but I am using the same colour twice — for the button background and for the button border, and it gets pretty ugly in the definition of the button border:

```
  border: var(--button-link-border, 1px solid var(--button-link-color, #{$blue}));
```

As Lea Verou says, "isn’t it so _horribly_ verbose?":

<img width="810" alt="image" src="https://user-images.githubusercontent.com/6834224/180332219-d9531bf0-4574-4a76-833d-59c9aef4ce8d.png">

So I took a hint from her 😄 I don't expect to use this often.

_(I realised by the way that I forgot to update the styles for the active and the disabled states of the button accordingly; so I am pushing a commit with an update now)_